api/types/network/network.go

```diff
@@ -60,6 +60,22 @@ type EndpointSettings struct {
 	MacAddress          string
 }
 
+// Task carries the information about one backend task
+type Task struct {
+	Name       string
+	EndpointID string
+	EndpointIP string
+	Info       map[string]string
+}
+
+// ServiceInfo represents service parameters with the list of service's tasks
+type ServiceInfo struct {
+	VIP          string
+	Ports        []string
+	LocalLBIndex int
+	Tasks        []Task
+}
```

Comment:
`ServiceInfo` and `Task` are shown only in the `verbose` case. Its not clear if multiple `responsess:` sections can be specified in `swagger.yml` file for different options of an API. Have to look into how its done in swagger. I will open an issue for this and get it done in a subsequent PR if thats ok.